app/components/agenda/move.hbs

```diff
@@ -0,0 +1,38 @@
+<label class="form__label">Verplaats agendapunt
+  <div class="grid">
+    <div class="col--6-12">
+      {{#power-select
```

Comment:
this should be converted to octane syntax, where possible (and I believe here it's very possible) it's better to avoid power select if we don't need advanced features (like search or typeahead). For a simple select, prefer the select element and element modifiers.

Reply:
It's been converted to octane so I will leave it for now
I'm not sure I agree with avoiding power-select as much as possible, I find simple selects annoying to work with and having two ways to deal with selects in the same component can be confusing.
Is powerselect really that heavy?